specials/SPC18_Matias_SpiesVsMercs.Altis/edit_me/description.ext

```diff
@@ -0,0 +1,30 @@
+// Mission settings - title, description, author etc.
+OnLoadName = "Spies vs Mercs";
+OnLoadMission = "SOMEONE MUST FIRST JOIN THE SPY SIDE BEFORE SOMEONE JOINS THE MERC SIDE. Spies will try to hack the laptops, mercs will try to protect them";
+author = "Matías Jackson, Synixe Contractors";
+
+synixe_type = 3; // 0: Contract, 1: Sub-Contract, 2: Training, 3: Special
+synixe_start_time = 0; // Start time of the mission.
+
+// Randomization
+disableRandomization[] = {}; // Disables randomization of units and vehicles based off of variable name or classname
+
+// Game end
+class CfgDebriefing {
+    class End1 {
+        title = "The Mercs win";
+        subtitle = "";
+        description = "All of the spies have been killed";
+        pictureBackground = "";
+        picture = "b_inf";
```

Comment:
It is going to show the blufor flag if the mercs win?

Reply:
Changed to Opfor